tests/e2e/utils/payments.js

```diff
@@ -275,10 +275,13 @@ export async function setupShortcodeCheckout( page, billingDetails = null ) {
 			'#billing_country',
 			billingDetails[ 'country_iso' ]
 		);
-		await page.selectOption(
-			'#billing_state',
-			billingDetails[ 'state_iso' ]
-		);
+
+		if ( billingDetails[ 'state' ] ) {
```

Comment:
I think this was intended to check for `state_iso`
```suggestion
		if ( billingDetails[ 'state_iso' ] ) {
```

Reply:
Either works but checking for `state_iso` makes more sense given the code below. Thanks!